packages/workflow/test/FilterParameter.test.ts

```diff
@@ -1035,163 +1044,187 @@ describe('FilterParameter', () => {
 				it.each([
 					{ left: ['foo', 'bar'], right: 'foo', expected: true },
 					{ left: ['foo', 'bar'], right: 'ba', expected: false },
-				])('array:contains($left,$right) === $expected', ({ left, right, expected }) => {
-					const result = executeFilter(
-						filterFactory({
-							conditions: [
-								{
-									id: '1',
-									leftValue: left,
-									rightValue: right,
-									operator: { operation: 'contains', type: 'array', rightType: 'any' },
-								},
-							],
-						}),
-					);
-					expect(result).toBe(expected);
-				});
+				] as unknown as Tests)(
```

Comment:
Would changing the definition of `FilterConditionValue` be wrong?

```ts
export type FilterConditionValue = {
	id: string;
	leftValue: NodeParameterValue | NodeParameterValue[];
	operator: FilterOperatorValue;
	rightValue: NodeParameterValue | NodeParameterValue[];
};
```

Reply:
i tried that and it made the editor throw errors. I want to revisit these types, but I don't think I want to do that in this PR.

Reply:
I tried `NodeParameterValue | NodeParameterValue[]` and `NodeParameterValue | NodeParameterValue[] | Date | DateTime`, and both break the editor.

Reply:
Created a follow up: https://github.com/n8n-io/n8n/pull/9930